src/gateway_gatt_char_wifi_ssid.erl

```diff
@@ -71,6 +71,8 @@ handle_signal(SignalID, Msg, State=#state{service_signal=SignalID}) ->
 
 -spec update_value(#state{}) -> #state{}.
 update_value(State=#state{service_path=CurrentPath}) ->
+    %% Start a scan to repopulate list of visible wifi services
+    connman:scan(wifi),
```

Comment:
I'm not sure why you're doing this here?

Reply:
The list gets clobbered when a hotspot connects to a Wi-Fi network.  Originally I put this scan in the `gateway_gatt_char_wifi_connect` module's `maybe_notify_value(State=#state{})` function but that resulted in the mobile app hanging on `connected` so I moved it here.  It could be that the mobile app continues to hang with the scan here.  I still need to test this.

Reply:
We've seen the list of available Wi-Fi networks get decimated when connecting to Wi-Fi for the first time during onboarding. Inserting a scan here repopulates the list after a hotspot comes online for the first time.